hbase_sql/src/main/java/com/splicemachine/derby/lifecycle/HEngineSqlEnv.java

```diff
@@ -68,6 +71,8 @@
     // numNodes is written to zookeeper by OlapServerMaster, so we have
     // to wait until the Olap Server comes up before getting numNodes from zookeeper.
     private static int MAX_EXECUTOR_CORES = -1;
+    // A value to use until the Olap Server comes up.
+    private static int DEFAULT_MAX_EXECUTOR_CORES = 8;
     private static int numSparkNodes = -1;
```

Comment:
I think this is not used, remove if so

Reply:
This is needed.  I ran into problems if I made getMaxExecutorCores non-retryable.  If we call it now before ZK data is there we can use the default.

Reply:
I meant the `numSparkNodes` field, I believe it's only written to but never used. Sorry for not being clear, I can see how my message was confusing!

Reply:
I see.  It might actually help for debugging to have a static field that tells us how many spark nodes there are (or how many we think there are), so if you don't mind I'd like to leave it in for now.

Reply:
Ok